packages/appstore/src/modules/onboarding/onboarding.scss

```diff
@@ -1,7 +1,7 @@
 .onboarding-wrapper {
     display: flex;
     flex-direction: column;
-    background-color: var(--text-less-prominent);
+    background-color: $color-grey-4;
```

Comment:
This color has a different equivalent on dark theme, you need to include both by using `var`